### PR TITLE
Lazylibrarian integration

### PR DESCRIPTION
**Pull request by philborman**
#### Database Migration
YES - XXXX | NO

#### Description
Allow prowlarr to add/change/delete lazylibrarian indexers

#### Todos
- [ ] Tests
- [ ] Translation Keys (./src/NzbDrone.Core/Localization/Core/en.json)
- [ ] [Wiki Updates](https://wiki.servarr.com)

#### Issues Fixed or Closed by this PR

* Fixes #XXXX

### CODE REVIEW

**Comment by bakerboy448** [comment]
there's no files changed here;  you're trying to PR from your develop branch

this may be a helpful read
https://docs.github.com/en/desktop

File: src/NzbDrone.Core/Applications/LazyLibrarian/LazyLibrarian.cs [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+using NzbDrone.Core.Configuration;
+using NzbDrone.Core.Indexers;
+
+namespace NzbDrone.Core.Applications.LazyLibrarian
+{
+    public class LazyLibrarian : ApplicationBase<LazyLibrarianSettings>
+    {
+        public override string Name => "LazyLibrarian";
+
+        private readonly ILazyLibrarianV3Proxy _lazylibrarianV3Proxy;
```
Comment (bakerboy448) [inline src/NzbDrone.Core/Applications/LazyLibrarian/LazyLibrarian.cs:19]: Why is v3 proxy being used?  LL does not have a V3

either drop V3 or replace with V1;  see what is done for Lidarr if V is included

**Review by bakerboy448 — CHANGES_REQUESTED** [review]

File: src/NzbDrone.Core/Applications/LazyLibrarian/LazyLibrarian.cs (hunk 1 above)
Comment (philborman) [reply src/NzbDrone.Core/Applications/LazyLibrarian/LazyLibrarian.cs:19]: Was changed here, but the changes are not appearing upstream. The LazyLibrarian code was copied from mylar which had the same v1 / v3 at the time :-)

Comment (philborman) [reply src/NzbDrone.Core/Applications/LazyLibrarian/LazyLibrarian.cs:19]: Unable to use git desktop as there is no Linux version.

**Comment by bakerboy448** [comment]
Closing for #469

**Pull request closed without merging**